packages/remix-runtime/README.md

```diff
@@ -0,0 +1,4 @@
+# Remix Server Runtime for Netlify
+
+The Remix Server Runtime for Netlify is a serverless edge runtime for [Remix](https://remix.run) apps. It's built on top
```

Comment:
Mind having a peek at this @stephmarie17? It's to the point, but just want to make sure about the wording.